app/scripts/objects/author.js

```diff
@@ -2,33 +2,43 @@ import _ from 'lodash';
 
 export default class {
 
-  constructor(data, subscriptionStore) {
+  constructor(token, author) {
 
-    if (_.isEmpty(data)) {
+    if (_.isEmpty(author)) {
       throw "Empty data object passed to Author constructor";
     }
 
-    this.id = data.id;
-    this.name = data.name;
-    this.github = data.github;
-    this.bio = data.bio;
-    this.image = data.image;
-    this.notifications = data.notifications;
-    this.subscriptionCount = data.subscriptions.length;
-    this.subscriptionStore = subscriptionStore;
+    this.token           = token ? token : null;
+    this.id              = author.id;
+    this.bio             = author.bio;
+    this.url             = author.url;
+    this.image           = author.image;
+    this.host            = author.host;
+    this.email           = author.email;
+    this.first_name      = author.first_name;
+    this.last_name       = author.last_name;
```

Comment:
Looks like we should be dealing with just displayName based on the spec now? Can probably do that in a different PR though.